observability/monitoring/config-map-reference-for-the-cluster-monitoring-operator.adoc

```diff
@@ -535,6 +537,10 @@ Appears in: link:#userworkloadconfiguration[UserWorkloadConfiguration]
 [options="header"]
 |===
 | Property | Type | Description
+|scrapeInterval|string|Configures the default interval between consecutive scrapes in case the `ServiceMonitor` or `PodMonitor` resource does not specify any value. The interval must be set between 5 seconds and 5 minutes. The value can be expressed in: seconds (for example `30s`), minutes (for example `1m`) or a mix of minutes and seconds (for example `1m30s`). The default value is `30s`.
```

Comment:
```suggestion
|scrapeInterval|string|Configures the default interval between consecutive scrapes in case the `ServiceMonitor` or `PodMonitor` resource does not specify any value. The interval must be set between 5 seconds and 5 minutes. The value can be expressed in seconds (for example `30s`), minutes (for example `1m`) or a mix of minutes and seconds (for example `1m30s`). The default value is `30s`.
```

Reply:
Same for other entries.